.github/workflows/build-test.yml

```diff
@@ -253,7 +253,7 @@ jobs:
       # Upload unit test coverage reports for later use in the "reports" job.
       - name: Upload unit test coverage reports
         uses: actions/upload-artifact@v2
-        if: always()
+        if: "(success() || failure()) && !contains(github.ref, 'refs/heads/release')"
```

Comment:
We won't want to upload coverage when we cancel CI.
Currently always means even if we cancel, it will run this task.
This takes a lot of time when cancelling jobs.